### DISCUSSION TITLE
Value colors

### DISCUSSION THREAD

**Discussion by cedschmidt-lab** [sbender9/WilhelmSKSupport#25] [Q&A]
Hi,

The app is supper cool and works very well. 
I just wonder wether it is possible to change the color of the value text. I saw it can change depending on the value but can we set a color manually?

Best regards,

**Comment by SCarns** [comment]
Should be able to. There's a color wheel in both Title Display & Units Display set ups that allow you to choose font, size, and color.

**Reply by cedschmidt-lab** [reply]
Yes for the title and the unit but I cannot find a color picker for the value itself
